src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -153,7 +162,11 @@ var CSSPropertyOperations = {
       }
       var styleValue = styles[styleName];
       if (__DEV__) {
-        warnValidStyle(styleName, styleValue);
+        var compositeComponentName = 'Component';
+        if (component && component._currentElement && component._currentElement._owner) {
+          compositeComponentName = component._currentElement._owner.getName();
+        }
+        warnValidStyle(styleName, styleValue, compositeComponentName);
```

Comment:
A few things…

1. Let's just do the work in warnValidStyle as opposed to doing it twice.
2. Let's refer to this as an `owner`. Owners are not necessarily Composite Components.
3. Let's do something like https://github.com/facebook/react/blob/c52265884a5e6726c712924ce074af5a01ae016e/src/renderers/dom/client/wrappers/LinkedValueUtils.js#L90-L98 - specifically the part where we just don't put anything if we don't have an owner. Falling back to "Component" here isn't really right - could actually have a component named Component which would be confusing or be in a top level render situation.